dockers/base-ipu/Dockerfile

```diff
@@ -17,14 +17,14 @@ FROM ubuntu:20.04
 LABEL maintainer="Lightning-AI <https://github.com/Lightning-AI>"
 
 ARG PYTHON_VERSION=3.9
-ARG CONDA_VERSION=4.9.2
+ARG PYTORCH_VERSION=1.9
```

Comment:
I'm confused. How was it possible to build the docker image with PyTorch 1.9 without `ARG PYTORCH_VERSION`? https://github.com/Lightning-AI/lightning/blob/b25275ccc27652b91d85d49b7bc220b37c921b54/.github/workflows/cicd-pytorch_dockers.yml#L169